fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexDeferredMaintenanceControl.java

```diff
@@ -31,7 +31,7 @@
  */
 public class IndexDeferredMaintenanceControl {
     private Set<Index> mergeRequiredIndexes = null;
-    private boolean autoMergeDuringCommit = true;
+    private boolean autoMergeDuringCommit = false;
```

Comment:
We should followup to improve the ux, and ensure that users can't accidentally never do merges.